src/ts-ast.ts

```diff
@@ -12,19 +12,21 @@
 // TODO Document classes better.
 // TODO Try to make serialization methods easier to read.
 
-export type Node =
-    Document|Namespace|Class|Interface|Function|Method|Type|Property;
+export type Node = Document|TopLevel|Method|Type|Property;
+
+// An AST node that can appear directly in a document or namespace.
+export type TopLevel = Namespace|Class|Interface|Function|ConstValue;
```

Comment:
I've seen the term `Declaration` used for things like this.

Reply:
Done.